Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -23,8 +24,8 @@ extension TabBarVisibility: RepresentableOptionType {
 
 /// The same style switch accessory view as in Static framework, except will not be recreated each time the Cell
 /// is configured, since it will be stored as is in `Row.Accessory.view`
+fileprivate typealias ValueChange = (Bool) -> Void
```

Comment:
One last random thing: why is this needed to be pulled out? 

You could just do this below in the `BoolRow` func: 
`... onValueChange: SwitchAccessoryView.ValueChange? = nil) -> Row {`

Reply:
Pushed back in. I was trying to keep it loosely coupled so that load is faster. But yeah it wont matter much.